src/docs/react-storybook/configurations/custom-webpack-config.js

```diff
@@ -4,6 +4,9 @@ export default {
   id: 'custom-webpack-config',
   title: 'Custom Webpack Config',
   content: stripIndent`
+    **NOTE: Storybook doesn't currently support webpack 2, so write your storybook webpack config to be
```

Comment:
Please include in one line, otherwise it renders like this:

<img width="695" alt="custom_webpack_config_-_react_storybook" src="https://cloud.githubusercontent.com/assets/488689/25476498/b6e1094a-2b7d-11e7-88b6-cf368b64fd12.png">